djoser/serializers.py

```diff
@@ -69,6 +70,8 @@ def validate(self, attrs):
                 raise serializers.ValidationError(self.error_messages['inactive_account'])
             return attrs
         else:
+            if User.objects.filter(**{User.USERNAME_FIELD: attrs.get(User.USERNAME_FIELD)}).first() is None:
```

Comment:
A better condition to check for would be:
`not User.objects.filter(**{User.USERNAME_FIELD: attrs.get(User.USERNAME_FIELD)}).exists()`